packages/gui/src/components/offers/OfferShareDialog.tsx

```diff
@@ -862,6 +917,87 @@ OfferShareHashgreenDialog.defaultProps = {
   onClose: () => {},
 };
 
+function OfferShareSpacescanDialog(props: OfferShareServiceDialogProps) {
+  const { offerRecord, offerData, testnet, onClose, open } = props;
+  const openExternal = useOpenExternal();
+  const [sharedURL, setSharedURL] = React.useState('');
+
+  function handleClose() {
+    onClose(false);
+  }
+
+  async function handleConfirm() {
+    const url = await postToSpacescan(offerData, testnet);
+    console.log(`Spacescan.io URL: ${url}`);
```

Comment:
for logging we are using "debug" npm library. you can use it or remove console.log

Reply:
Thanks, I added a new `chia-gui:offers` log outlet. I'd like to start doing more GUI logging, so this is a good reminder.